projects/npm-tools/packages/npm-scripts/src/sass/build.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const fs = require('fs');
+const globby = require('globby');
+const r2 = require('liferay-r2');
+const path = require('path');
+const sass = require('sass');
+
+const log = require('../utils/log');
+
+const SASS_EXCLUDE = [
+	'**/_*.scss',
```

Comment:
I thought Sass did this by default... 🤔

Based on https://sass-lang.com/guide:

> Partials
>
> You can create partial Sass files that contain little snippets of CSS that you can include in other Sass files. [...] A partial is a Sass file named with a leading underscore. [...] **The underscore lets Sass know that the file is only a partial file and that it should not be generated into a CSS file**. Sass partials are used with the @use rule.

Reply:
@jbalsas this was also specified in our java implementation. I haven't tested it without it, but I think it seems reasonable to make it an explicit exclude.

Reply:
Just tested it, if we don't include this here then the partials files will build into their own css files